zstacklib/zstacklib/test/lvm/test_config_lvm_filter.py

```
@@ -0,0 +1,307 @@
+            self.assertIn('"a|^%s$|"' % disk, new,
+                          "missing accept entry for %s" % disk)
+        self.assertIn('"r|.*|"', new)
+
+        for raw in original.splitlines():
+            if raw.lstrip().startswith("#") and ("filter" in raw or "global_filter" in raw):
+                self.assertIn(raw, new,
+                              "lost stock comment line: %r" % raw)
+
+        for noisy in ("# Example:", "# filter = [", "# global_filter = ["):
+            self.assertTrue(any(noisy in l and l.lstrip().startswith("#")
+                                for l in new.splitlines()),
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)